substrate/frame/support/src/traits/voting.rs

```diff
@@ -85,6 +85,7 @@ pub trait Polling<Tally> {
 	type Index: Parameter + Member + Ord + PartialOrd + Copy + HasCompact + MaxEncodedLen;
 	type Votes: Parameter + Member + Ord + PartialOrd + Copy + HasCompact + MaxEncodedLen;
 	type Class: Parameter + Member + Ord + PartialOrd + MaxEncodedLen;
+	type Who;
```

Comment:
Same.

Also while there is not yet docs, we should start :D Maybe add a doc explaining this.